bindgen/src/JSONBuilder.ts

```diff
@@ -49,6 +46,13 @@ function isField(mem: DeclarationStatement) {
   return mem.kind == NodeKind.FIELDDECLARATION;
 }
 
+function isPayable(func: FunctionDeclaration): boolean {
```

Comment:
Does this work yet? It doesn't seem to be exported or used anywhere. Maybe we can just remove it for now?

Reply:
I was going to make a PR to make it work and this would be a stepping stone to get there.